scipy/stats/_continuous_distns.py

```diff
@@ -678,33 +677,62 @@ def _pdf(self, x, a, b):
         # beta.pdf(x, a, b) = ------------------------------------
         #                              gamma(a)*gamma(b)
         with np.errstate(over='ignore'):
-            return _boost._beta_pdf(x, a, b)
+            return scu._beta_pdf(x, a, b)
 
     def _logpdf(self, x, a, b):
         lPx = sc.xlog1py(b - 1.0, -x) + sc.xlogy(a - 1.0, x)
         lPx -= sc.betaln(a, b)
         return lPx
 
     def _cdf(self, x, a, b):
-        return _boost._beta_cdf(x, a, b)
+        return sc.betainc(a, b, x)
 
     def _sf(self, x, a, b):
-        return _boost._beta_sf(x, a, b)
+        return sc.betaincc(a, b, x)
 
     def _isf(self, x, a, b):
-        with np.errstate(over='ignore'):  # see gh-17432
-            return _boost._beta_isf(x, a, b)
+        return sc.betainccinv(a, b, x)
 
     def _ppf(self, q, a, b):
-        with np.errstate(over='ignore'):  # see gh-17432
-            return _boost._beta_ppf(q, a, b)
+        return scu._beta_ppf(q, a, b)
 
     def _stats(self, a, b):
+        a_plus_b = np.add(a, b)
+        _beta_mean = np.divide(a, a_plus_b)
+        _beta_variance = np.divide(
+            np.multiply(a, b),
+            np.multiply(
+                np.square(a_plus_b),
+                np.add(a_plus_b, 1)
+            )
+        )
+        _beta_skewness = np.divide(
+            np.multiply(
+                np.multiply(2, np.subtract(b, a)),
+                np.sqrt(np.add(a_plus_b, 1))
+            ),
+            np.multiply(
+                np.add(a_plus_b, 2),
+                np.sqrt(np.multiply(a, b))
+            )
+        )
+        _beta_kurtosis_excess = np.divide(
+            np.multiply(6, np.subtract(
+                np.multiply(np.square(np.subtract(a, b)),
+                            np.add(a_plus_b, 1)),
+                np.multiply(np.multiply(a, b),
+                            np.add(a_plus_b, 2))
+            )),
```

Comment:
Is there a reason this is written like this instead of with operators? It's bringing back flashbacks to when I used to toy around with Lisp programming. Now, like then, I have a hard time parsing these mentally when written in prefix form. Would you consider changing this, and elsewhere in the file?

Reply:
Sure. I would. 

> Is there a reason this is written like this instead of with operators?

Just thought calling `np` APIs directly would be better. But I think that shouldn't make much of a difference.

Reply:
I see. I think the performance difference, if any, would be negligible -- so better to go with what's more readable to most people IMO.

Reply:
Addressed.